Create CITATION.cff

aubertc opened the pull request:
Tested valid w.r.t. specification at https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md

aubertc committed:
Create CITATION.cff

aubertc committed:
Corrected CITATION.cff

Added [members of the original team](https://coqtail.github.io/team.html)

aubertc committed:
Corrected CITATION.cff

Added [members from the original team](https://sourceforge.net/p/coqtail/_members/)

aubertc commented:
Ok thanks @gallais , I've tried to fix that, but cannot access the members of the "Former team (COQUILLE project)" as [the page seems to be in error](http://graal.ens-lyon.fr/coquille/index.php?page=team) and [the archives captured the 404 error message](https://web.archive.org/web/20250215195734/https://graal.ens-lyon.fr/coquille/index.php?page=team).

jmadiot commented:
Hi, thanks. The list of authors in the README has been kept up-to-date, as far as I can tell. Can you update the PR so that they are the same? Also can you pick the maintainer as contact?

aubertc committed:
Improved CITATION.cff

Added Amaury Pouly's full name.

aubertc committed:
Fixed contact in CITATION.cff

Contact is now maintainer.

jmadiot committed:
Update CITATION.cff

added Iooss, Capdevielle (worked on Hierarchy) Rabie (on Reals), removed others whose work was not included

aubertc commented:
Nope, cffconvert says it's still valid.
